playwright.config.ts

```diff
@@ -0,0 +1,23 @@
+import { defineConfig, devices } from '@playwright/test'
+export default defineConfig({
+  testDir: './e2e/tests',
+  fullyParallel: true,
+  forbidOnly: !(process.env.CI == null),
+  retries: process.env.CI != null ? 2 : 0,
+  workers: process.env.CI != null ? 1 : undefined,
+  reporter: 'html',
+  use: {
+    baseURL: 'https://deepbluecltd.github.io/',
```

Comment:
@qajay97 - is the test running against the hosted instance at that URL?  We need the test to run against the code for the current branch.

Sorry, I have no idea how to do that, but I hope Google is your friend.

Reply:
Changed the baseUrl to include `/RCO` path